Description: accept newer major version of crate fs-err
Author: Blair Noctis <ncts@debian.org>
Last-Update: 2025-08-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -41,7 +41,7 @@
 flate2 = { version = "1.0", optional = true, default-features = false, features = [
   "rust_backend",
 ] }
-fs-err = "2.11"
+fs-err = ">= 2.11, <= 3"
 futures = "0.3"
 gzp = { version = "0.11.3", default-features = false, features = [
   "deflate_rust",
